database/backend/architecture.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "Milan Jovanović",
+        "description": "Milan Jovanović makes videos about software engineering and software architecture, distributed systems, Domain-Driven Design.",
```

Comment:
Great start @sinannar! However, I think it would be beneficial to mention how Milan's course differs from other ones. It'll pique people's interest a bit more. 😄

Reply:
put more descrip tion milan, but moved it under youtube as requested. Put azure, aws and gcp cloud to architecture section.